front/components/assistant_builder/types.ts

```diff
@@ -147,3 +161,90 @@ export function getDefaultAssistantState(): AssistantBuilderState {
     },
   };
 }
+
+export function getDefaultRetrievalSearchActionConfiguration() {
```

Comment:
no explicit return allows to have `configuration` be properly typed as the right variant of the enum